ggshield/sca/output/schemas.py

```diff
@@ -0,0 +1,65 @@
+from dataclasses import dataclass, field
+from typing import List, Optional, Type, cast
+
+import marshmallow_dataclass
+from pygitguardian.models import Base, BaseSchema, FromDictMixin
+
+
+@dataclass
+class SCAVulnerability(Base, FromDictMixin):
+    severity: str
+    summary: str
+    cve_ids: List[str] = field(default_factory=list)
+    created_at: Optional[str] = None
+    fixed_version: Optional[str] = None
+
+
+SCAVulnerabilitySchema = cast(
```

Comment:
why do you need this additional variable?

Reply:
Yes, some code in pygg does declare a FooSchema variable, that is kept because removing it would break API compatibility, see ServerMetadata for a recent example which does what Achille suggests.